app/src/main/res/drawable/ic_star.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Vector drawables are a little performance reducer but i think it will be compensated by the usage of colorFilter, tint used in making the drawable icon yellow! 
So we get the benefit of the points  mentioned by rahul!